### ISSUE TITLE
HEAD zimwriterfs/zimlib seems to have broken suggestions

### ISSUE THREAD

**Issue by kelson42** [openzim/zimwriterfs#43]
Using https://download.kiwix.org/zim/other/installgentoo_en_all_2018-05.zim which has been created with most recent zimwriterfs (and last official release of kiwix-serve).

http://download.kiwix.org:8080/search?content=installgentoo_en_all_2018-05&pattern=Cryptocurrency
gives "not found"

http://download.kiwix.org:8080/search?content=installgentoo_en_all_2018-03&pattern=Cryptocurrency
gives the article at http://download.kiwix.org:8080/installgentoo_en_all_2018-03/A/Cryptocurrency.html

Basically that means that the suggestion system does not work anymore.

**Comment by kelson42** [comment]
I see good chances, that this is related to this another bug https://github.com/kiwix/kiwix-tools/issues/193

**Comment by mgautierfr** [comment]
Fixed with correction of kiwix/kiwix-tools#193
